monitoring/iostats_context.cc

```diff
@@ -15,6 +15,7 @@ namespace ROCKSDB_NAMESPACE {
 static IOStatsContext iostats_context;
 #else
 thread_local IOStatsContext iostats_context;
+thread_local bool enable_iostats = true;
```

Comment:
Is there a reason to add this here rather than make it part of the IOStatsContext?  Wouldn't there be better performance if "enabled" was inside of IOStatsContext (accessing one thread local instead of two...)?